active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
@jairideout , I would prefer `interval_metadata` over `feature_metadata`, because it is more specific and self-evident to have `interval` in the name and corresponding well to `positional_metadata`.

I can't think of other use cases for `interval_metadata` besides associated with `Sequence`. a possible use case is in the time series study, with `interval` to store time interval and `metadata` to store sample properties? But this is only hypothetical.